### PR TITLE
Add Uploader

### PR DESCRIPTION
**Pull request by kj13ennett**

### CODE REVIEW

File: app/src/main/java/com/pixlee/pixleeandroidsdk/SampleActivity.java [hunk 1]
```diff
@@ -176,7 +176,7 @@ public void onLoadMore(int page, int totalItemsCount, RecyclerView view) {
      */
     private void createAlbum() {
         Context c = this.getApplicationContext();
-        PXLClient.initialize("196i8ZzIAhKU8dO2kDe");
+        PXLClient.initialize("196i8ZzIAhKU8dO2kDe", "66f3c350bc162c143eecd41711");
```
Comment (firetix) [inline app/src/main/java/com/pixlee/pixleeandroidsdk/SampleActivity.java]: Should remove the secret_key and cleanup the commits with them.

Comment (jchen123) [reply app/src/main/java/com/pixlee/pixleeandroidsdk/SampleActivity.java]: oh my god lol bfg this please this is a public repo

**Comment by kj13ennett** [comment]
@firetix @jchen123 ready for review

File: pixleesdk/src/main/java/com/pixlee/pixleesdk/PXLClient.java [hunk 2]
```diff
@@ -161,6 +188,88 @@ public void onErrorResponse(VolleyError error) {
...
+        }
+        String paramString = String.format("%s=%s", KeyApiKey, apiKey);
+        String finalUrl = String.format("%s/%s?%s", url, requestPath, paramString);
+
+        Log.d(TAG, "POST");
+
+        Log.d(TAG, finalUrl);
+
+
+        final String requestBody = body.toString().replace("\\/", "/" );
```
Comment (firetix) [inline pixleesdk/src/main/java/com/pixlee/pixleesdk/PXLClient.java]: so you had to do this as well??

File: app/src/main/java/com/pixlee/pixleeandroidsdk/SampleActivity.java [hunk 3]
```diff
@@ -176,7 +176,9 @@ public void onLoadMore(int page, int totalItemsCount, RecyclerView view) {
      */
     private void createAlbum() {
         Context c = this.getApplicationContext();
+
         PXLClient.initialize("196i8ZzIAhKU8dO2kDe");
```
Comment (jchen123) [inline app/src/main/java/com/pixlee/pixleeandroidsdk/SampleActivity.java]: dumb question, you can fire this function without the 2nd param?

Comment (firetix) [inline app/src/main/java/com/pixlee/pixleeandroidsdk/SampleActivity.java]: is secret_key optional parameters?

File: pixleesdk/src/main/java/com/pixlee/pixleesdk/PXLClient.java (hunk 2 above)
Comment (kj13ennett) [reply pixleesdk/src/main/java/com/pixlee/pixleesdk/PXLClient.java]: yup

File: app/src/main/java/com/pixlee/pixleeandroidsdk/SampleActivity.java (hunk 3 above)
Comment (kj13ennett) [reply app/src/main/java/com/pixlee/pixleeandroidsdk/SampleActivity.java]: yes, I have 2 constructors in PXLClient

Comment (kj13ennett) [reply app/src/main/java/com/pixlee/pixleeandroidsdk/SampleActivity.java]: yes, I have 2 constructors in PXLClient

File: README.md [hunk 4]
```diff
@@ -70,6 +70,24 @@ Each successive call of `loadNextPageOfPhotos` will load the next page of photos
...
+
+If you wish to build your application with photo uploading capabilities, you must initialize the `PXLClient` with an additional parameter - your Pixlee secret key. You must set the secret key along with the API key when you call the static method initialize:
+
+```
+#!java
+
+PXLClient.initialize(<API KEY>, <SECRET KEY>);
+```
+
+Now wen you want to upload a photo in your application, simply call the `uploadImage` method of the PXLAlbum object you are using.  This woul look something like this:
```
Comment (jchen123) [inline README.md]: woul -> would

**Pull request merged**